views/partials/top.handlebars

```diff
@@ -1,15 +1,49 @@
-<section class="hero is-medium is-primary">
-<div class="hero-body">
-  <div class="container has-text-centered">
-    <h3 class="subtitle">
-      Programe-se para aprender!
-    </h3>
-    <h1 class="title">
-      &#60;/CONHECIMENTO LIVRE&#62;
-    </h1>
-    <h3 class="subtitle">
-      Cursos de programação gratuitos.
-    </h3>
+<section class="hero pag-inicial is-medium ">
+  {{> header}}
+  <div class="hero-body">
+    <div class="container">
+      <h1 class="title is-1">
+        Conhecimento <br>para todos!</br>
```

Comment:
br deve ser utilizado apenas dentro de parágrafos `<br>`

Ref: https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br